schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                    "type": "array"
+                },
+                "adminTab": {
+                    "description": "Adds a Tab which can be shown in admin adapter",
+                    "type": "object",
+                    "required": ["fa-icon", "link", "name"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "fa-icon": {
+                            "description": "Font-Awesome icon name for TAB",
+                            "type": "string"
+                        },
+                        "ignoreConfigUpdate": {
+                            "description": "Do not update config TAB if configuration changed (to enable configure settings in TAB)",
+                            "type": "boolean"
+                        },
+                        "link": {
+                            "description": "Link for iframe in the TAB. You can use parameters replacement like this: \"http://%ip%:%port%\". IP will be replaced with host IP. \"port\" will be extracted from native.port.",
+                            "type": "string"
+                        },
+                        "name": {
+                            "description": "Name of TAB (multilingual object) in admin",
+                            "type": "object",
+                            "required": ["en"],
+                            "patternProperties": {
+                                "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
+                        },
+                        "singleton": {
+                            "description": "If true, only one TAB for all instances will be shown.",
+                            "type": "boolean"
+                        }
+                    }
+                },
+                "controller": {
+                    "description": "Only used by js-controller",
+                    "type": "boolean"
+                },
+                "allowInit": {
+                    "description": "allow for \"scheduled\" adapter to be called \"not in the time schedule\", if settings changed or adapter started. Or allow scheduled adapter start once after configuration changed and then by schedule.",
+                    "type": "boolean"
+                },
+                "availableModes": {
+                    "description": "values for common.mode if more than one mode is possible",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
```

Comment:
We can be more specific here. If unsure, I have them in the type definitions.

Reply:
same as mode of course